assets/js/admin/orders/index.js

```diff
@@ -1,12 +1,34 @@
 /**
  * Internal dependencies
  */
-import './order-overview';
+import OrderOverview from './order-overview';
 import './order-details';
-
 import { jQueryReady } from 'utils/jquery.js';
 
 jQueryReady( () => {
+	// Order Overview.
```

Comment:
@robincornett I think since you can change the Customer after the Order is created it shouldn't be required. But I'm open to requiring it.

Can you add any specific errors and steps you're taking to get the errors to https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7464